.travis.yml

```diff
@@ -30,33 +31,20 @@ stages:
 
 jobs:
   include:
-
-# ---------------------------------------
-# Linter
-# ---------------------------------------
+    # ---------------------------------------
+    # Linter
+    # ---------------------------------------
     - stage: lint
-      name: "Linter"
+      name: 'Linter'
       script:
         - npm install
         - npm run lint
 
-# ---------------------------------------
-# Unit Testing
-# ---------------------------------------
-    - stage: tests
-      name: "Unit Tests"
```

Comment:
I think that unit tests are still present and relevant in 2-x, shouldn't we keep them?